### ISSUE TITLE
[FX] Support call_method in NormalizeArgs

### ISSUE THREAD

**Issue by jfix71** [pytorch/pytorch#71715]
### 🚀 The feature, motivation and pitch

Currently NormalizeArgs only supports normalizing `call_function`s and `call_module`s. Ideally it would support `call_method`s as well. Example code:

```python
import torch
import torch.fx

class TestModule(torch.nn.Module):
    def forward(self, x):
        return x.flatten(), torch.flatten(x)

mod = TestModule()
gm = torch.fx.symbolic_trace(mod)
print(gm.graph)
```
Output shows flatten args for `call_method` and `call_function` not normalized with standard FX tracing (all kwargs usage or default values), as expected:
```
graph():
    %x : [#users=2] = placeholder[target=x]
    %flatten : [#users=1] = call_method[target=flatten](args = (%x,), kwargs = {})
    %flatten_1 : [#users=1] = call_function[target=torch.flatten](args = (%x,), kwargs = {})
    return (flatten, flatten_1)
```

Now when we normalize we see that the `call_function` is normalized with all kwargs and default values, but the `call_method` is not.
```python
from torch.fx.experimental.normalize import NormalizeArgs
gm = NormalizeArgs(gm).transform()
print(gm.graph)
```
```
graph():
    %x : [#users=2] = placeholder[target=x]
    %flatten : [#users=1] = call_method[target=flatten](args = (%x,), kwargs = {})
    %flatten_1 : [#users=1] = call_function[target=torch.flatten](args = (), kwargs = {input: %x, start_dim: 0, end_dim: -1})
    return (flatten, flatten_1)
```
